File: app/src/main/java/com/izyver/gati/network/GatiApi.java
package com.izyver.gati.network;

import com.izyver.gati.model.entity.ScheduleObject;

import retrofit2.Call;
import retrofit2.http.GET;

public interface GatiApi {

    @GET("api/schedule")
    Call<ScheduleObject> getSchedulers();
}

File: app/src/main/java/com/izyver/gati/data/network/GatiApi.kt
package com.izyver.gati.data.network

import com.izyver.gati.old.model.entity.ScheduleObject

import retrofit2.Call
import retrofit2.http.GET

interface GatiApi {
    @GET("api/schedule")
    fun getSchedulers(): Call<ScheduleObject>
}
